src/ngraph_rewrite_for_tracking.h

```diff
@@ -22,8 +22,18 @@ namespace tensorflow {
 
 namespace ngraph_bridge {
 
-Status RewriteForTracking(Graph* graph);
-
+Status RewriteForTracking(Graph* graph, int graph_id);
+Status ReplaceNGraphAssign(Graph* graph, Node* node, Node** replacement,
+                           std::string node_new_name, bool just_looking,
+                           bool outputs_ng_supported);
+Status ReplaceNGraphVariable(Graph* graph, Node* node, Node** replacement,
+                             std::string node_new_name, bool just_looking,
+                             bool outputs_ng_supported);
+Status ReplaceNGraphApplyGradientDescent(Graph* graph, Node* node,
+                                         Node** replacement,
+                                         std::string node_new_name,
+                                         bool just_looking,
+                                         bool outputs_ng_supported);
```

Comment:
Just to clarify, these function prototypes are defined in `src/ngraph_rewrite_for_tracking.h` but are actually defined in `src/enable_variable_ops/ngraph_rewrite_for_tracking.cc` and not `src/ngraph_rewrite_for_tracking.cc`, right?

Reply:
yes

Reply:
yes